src/wallet/wallet.cpp

```diff
@@ -3437,6 +3437,45 @@ void CWallet::ListLockedCoins(std::vector<COutPoint>& vOutpts)
     }
 }
 
+
+// Note Locking Operations
+
+void CWallet::LockNote(JSOutPoint& output)
+{
+    AssertLockHeld(cs_wallet); // setLockedNotes
+    setLockedNotes.insert(output);
+}
+
+void CWallet::UnlockNote(JSOutPoint& output)
+{
+    AssertLockHeld(cs_wallet); // setLockedNotes
+    setLockedNotes.erase(output);
+}
+
+void CWallet::UnlockAllNotes()
+{
+    AssertLockHeld(cs_wallet); // setLockedNotes
+    setLockedNotes.clear();
+}
+
+bool CWallet::IsLockedNote(uint256 hash, size_t js, uint8_t n) const
+{
+    AssertLockHeld(cs_wallet); // setLockedNotes
+    JSOutPoint outpt(hash, js, n);
+
+    return (setLockedNotes.count(outpt) > 0);
+}
+
+void CWallet::ListLockedNotes(std::vector<JSOutPoint>& vOutpts)
+{
+    AssertLockHeld(cs_wallet); // setLockedNotes
+    for (std::set<JSOutPoint>::iterator it = setLockedNotes.begin();
+         it != setLockedNotes.end(); it++) {
```

Comment:
We tend to use two indentation levels in this situation to distinguish from the inner content.

Reply:
I just put it all on one line.